site2/website/blog/2022-04-08-Apache-Pulsar-2-9-2.md

```diff
@@ -0,0 +1,131 @@
+---
+author: gaoran10, Anonymitaet
+title: What’s New in Apache Pulsar 2.9.2
+---
+
+The Apache Pulsar community releases version 2.9.2! 60 contributors provided improvements and bug fixes that delivered 317 commits.
+
+Highlights of this release are as below:
+
+- Transactions performance test tool is available. [PR-11933](https://github.com/apache/pulsar/pull/11933)
+  
+- Brokers decrease the number of unacked messages. [PR-13383](https://github.com/apache/pulsar/pull/13383)
+  
+- Readers continue to read data from the compacted ledgers. [PR-13629](https://github.com/apache/pulsar/pull/13629)
+
+This blog walks through the most noteworthy changes grouped by the affected functionalities. For the complete list including all features, enhancements, and bug fixes, check out the [Pulsar 2.9.2 Release Notes](https://github.com/apache/pulsar/releases/tag/v2.9.2).
+
+# Notable bug fixes and enhancements
+
+### Readers continue to read data from the compacted ledgers. [PR-13629](https://github.com/apache/pulsar/pull/13629)
+
+#### Issue
+
+Previously, when topics were unloaded, some data was lost to be read by readers if they have consumed some messages from some compacted ledgers.
+
+#### Resolution
+
+Rewound the reader cursor to the next message of the mark delete position if `readCompacted = true`.
+
+### Brokers decrease the number of unacked messages. [PR-13383](https://github.com/apache/pulsar/pull/13383)
+
+#### Issue
+
+Previously, brokers did not decrease the number of unacked messages if batch ack was enabled. Consequently, consumers were blocked if they reached `maxUnackedMessagesPerConsumer` limit.
+
+#### Resolution
+
+Decreased the number of unacked messages when `individualAckNormal` was called.
+
+### Chunked messages can be queried through Pulsar SQL. [PR-12720](https://github.com/apache/pulsar/pull/12720)
+
+#### Issue
+
+Previously, chunked messages could not be queried through Pulsar SQL.
+
+#### Resolution
+
+Added a chunked map to support this enhancement.
```

Comment:
```suggestion
Add a chunked message map in PulsarRecordCursor to maintain incomplete chunked messages, if one chunked message was received completely, it will be offered in the message queue to wait for deserialization. 
```